src/fidesops/schemas/connection_configuration/connection_secrets_snowflake.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional, List
+
+from fidesops.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class SnowflakeSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed to connect to Snowflake"""
+
+    user_login_name: str
+    password: str
+    account_identifier: str  # Do not include the snowflakecomputing.com domain name as part of your account identifier.
```

Comment:
We've not discussed this as a team yet, but providing example data in the schemas could be a really good way around having comments like this: https://fastapi.tiangolo.com/tutorial/schema-extra-example/. We should consider doing this for all schemas that are referenced in the API docs, will add it to the list of retro talking points.

Reply:
ah yes that would be useful